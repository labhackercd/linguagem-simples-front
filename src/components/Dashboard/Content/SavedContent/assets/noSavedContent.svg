<svg width="216" height="76" viewBox="0 0 216 76" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M7.01367 63.0059H8.70898L5.20898 73H3.48633L0 63.0059H1.68164L3.7666 69.2197C3.87598 69.516 3.99219 69.8988 4.11523 70.3682C4.23828 70.833 4.31803 71.1794 4.35449 71.4072C4.41374 71.0609 4.50488 70.6598 4.62793 70.2041C4.75098 69.7484 4.85124 69.4111 4.92871 69.1924L7.01367 63.0059ZM16.5635 69.2061C16.5635 70.4411 16.2467 71.4049 15.6133 72.0977C14.9798 72.7904 14.098 73.1367 12.9678 73.1367C12.2614 73.1367 11.637 72.9772 11.0947 72.6582C10.5524 72.3392 10.1354 71.8812 9.84375 71.2842C9.55208 70.6872 9.40625 69.9945 9.40625 69.2061C9.40625 67.9801 9.7207 67.0231 10.3496 66.335C10.9785 65.6468 11.8649 65.3027 13.0088 65.3027C14.1025 65.3027 14.9684 65.6559 15.6064 66.3623C16.2445 67.0641 16.5635 68.012 16.5635 69.2061ZM11.0605 69.2061C11.0605 70.9515 11.7054 71.8242 12.9951 71.8242C14.2712 71.8242 14.9092 70.9515 14.9092 69.2061C14.9092 67.4788 14.2666 66.6152 12.9814 66.6152C12.307 66.6152 11.8171 66.8385 11.5117 67.2852C11.2109 67.7318 11.0605 68.3721 11.0605 69.2061ZM21.4648 73.1367C20.321 73.1367 19.4505 72.804 18.8535 72.1387C18.2611 71.4688 17.9648 70.5094 17.9648 69.2607C17.9648 67.9893 18.2747 67.0117 18.8945 66.3281C19.5189 65.6445 20.4189 65.3027 21.5947 65.3027C22.3923 65.3027 23.11 65.4508 23.748 65.7471L23.2627 67.0391C22.5837 66.7747 22.0231 66.6426 21.5811 66.6426C20.2731 66.6426 19.6191 67.5107 19.6191 69.2471C19.6191 70.0947 19.7809 70.7327 20.1045 71.1611C20.4326 71.585 20.9111 71.7969 21.54 71.7969C22.2555 71.7969 22.9323 71.6191 23.5703 71.2637V72.665C23.2832 72.8337 22.9756 72.9544 22.6475 73.0273C22.3239 73.1003 21.9297 73.1367 21.4648 73.1367ZM28.6494 73.1367C27.4736 73.1367 26.5531 72.7949 25.8877 72.1113C25.2269 71.4232 24.8965 70.4775 24.8965 69.2744C24.8965 68.0394 25.2041 67.0687 25.8193 66.3623C26.4346 65.6559 27.2799 65.3027 28.3555 65.3027C29.3535 65.3027 30.1419 65.6058 30.7207 66.2119C31.2995 66.818 31.5889 67.652 31.5889 68.7139V69.582H26.5508C26.5736 70.3158 26.7718 70.8809 27.1455 71.2773C27.5192 71.6693 28.0456 71.8652 28.7246 71.8652C29.1712 71.8652 29.5859 71.8242 29.9688 71.7422C30.3561 71.6556 30.7708 71.5143 31.2129 71.3184V72.624C30.821 72.8109 30.4245 72.943 30.0234 73.0205C29.6224 73.098 29.1644 73.1367 28.6494 73.1367ZM28.3555 66.5195C27.8451 66.5195 27.4349 66.6813 27.125 67.0049C26.8197 67.3285 26.6374 67.8001 26.5781 68.4199H30.0098C30.0007 67.7956 29.8503 67.3239 29.5586 67.0049C29.2669 66.6813 28.8659 66.5195 28.3555 66.5195ZM29.9277 64.5166C29.3672 64.1839 28.8477 63.7738 28.3691 63.2861C27.8997 63.7555 27.387 64.1657 26.8311 64.5166H25.751V64.3457C26.6214 63.4434 27.2002 62.7529 27.4873 62.2744H29.2646C29.5518 62.7757 30.1351 63.4661 31.0146 64.3457V64.5166H29.9277ZM41.7812 73L41.46 71.9473H41.4053C41.0407 72.4076 40.6738 72.722 40.3047 72.8906C39.9355 73.0547 39.4616 73.1367 38.8828 73.1367C38.14 73.1367 37.5589 72.9362 37.1396 72.5352C36.7249 72.1341 36.5176 71.5667 36.5176 70.833C36.5176 70.0537 36.807 69.4658 37.3857 69.0693C37.9645 68.6729 38.8464 68.4564 40.0312 68.4199L41.3369 68.3789V67.9756C41.3369 67.4925 41.223 67.1325 40.9951 66.8955C40.7718 66.654 40.4232 66.5332 39.9492 66.5332C39.5618 66.5332 39.1904 66.5902 38.835 66.7041C38.4795 66.818 38.1377 66.9525 37.8096 67.1074L37.29 65.959C37.7002 65.7448 38.1491 65.583 38.6367 65.4736C39.1243 65.3597 39.5846 65.3027 40.0176 65.3027C40.9792 65.3027 41.7038 65.5124 42.1914 65.9316C42.6836 66.3509 42.9297 67.0094 42.9297 67.9072V73H41.7812ZM39.3887 71.9062C39.972 71.9062 40.4391 71.7445 40.79 71.4209C41.1455 71.0928 41.3232 70.6348 41.3232 70.0469V69.3906L40.3525 69.4316C39.596 69.459 39.0446 69.5866 38.6982 69.8145C38.3564 70.0378 38.1855 70.3818 38.1855 70.8467C38.1855 71.1839 38.2858 71.446 38.4863 71.6328C38.6868 71.8151 38.9876 71.9062 39.3887 71.9062ZM46.7783 73H45.1719V65.4395H46.7783V73ZM45.0762 63.4365C45.0762 63.1494 45.1536 62.9284 45.3086 62.7734C45.4681 62.6185 45.6937 62.541 45.9854 62.541C46.2679 62.541 46.4867 62.6185 46.6416 62.7734C46.8011 62.9284 46.8809 63.1494 46.8809 63.4365C46.8809 63.71 46.8011 63.9264 46.6416 64.0859C46.4867 64.2409 46.2679 64.3184 45.9854 64.3184C45.6937 64.3184 45.4681 64.2409 45.3086 64.0859C45.1536 63.9264 45.0762 63.71 45.0762 63.4365ZM55.7334 73H54.1201V68.3516C54.1201 67.7682 54.0016 67.333 53.7646 67.0459C53.5322 66.7588 53.1608 66.6152 52.6504 66.6152C51.9714 66.6152 51.4746 66.8158 51.1602 67.2168C50.8457 67.6178 50.6885 68.29 50.6885 69.2334V73H49.082V65.4395H50.3398L50.5654 66.4307H50.6475C50.8753 66.0706 51.1989 65.7926 51.6182 65.5967C52.0374 65.4007 52.5023 65.3027 53.0127 65.3027C54.8265 65.3027 55.7334 66.2256 55.7334 68.0713V73ZM60.5186 73.1367C59.5752 73.1367 58.8392 72.7949 58.3105 72.1113C57.7819 71.4277 57.5176 70.4684 57.5176 69.2334C57.5176 67.9938 57.7842 67.0299 58.3174 66.3418C58.8551 65.6491 59.598 65.3027 60.5459 65.3027C61.5394 65.3027 62.2959 65.6696 62.8154 66.4033H62.8975C62.82 65.861 62.7812 65.4326 62.7812 65.1182V62.3633H64.3945V73H63.1367L62.8564 72.0088H62.7812C62.2663 72.7607 61.512 73.1367 60.5186 73.1367ZM60.9492 71.8379C61.61 71.8379 62.0908 71.6533 62.3916 71.2842C62.6924 70.9105 62.8473 70.3066 62.8564 69.4727V69.2471C62.8564 68.2946 62.7015 67.6178 62.3916 67.2168C62.0817 66.8158 61.5964 66.6152 60.9355 66.6152C60.3704 66.6152 59.9352 66.8454 59.6299 67.3057C59.3245 67.7614 59.1719 68.4131 59.1719 69.2607C59.1719 70.0993 59.32 70.7396 59.6162 71.1816C59.9124 71.6191 60.3568 71.8379 60.9492 71.8379ZM71.4219 73L71.1006 71.9473H71.0459C70.6813 72.4076 70.3145 72.722 69.9453 72.8906C69.5762 73.0547 69.1022 73.1367 68.5234 73.1367C67.7806 73.1367 67.1995 72.9362 66.7803 72.5352C66.3656 72.1341 66.1582 71.5667 66.1582 70.833C66.1582 70.0537 66.4476 69.4658 67.0264 69.0693C67.6051 68.6729 68.487 68.4564 69.6719 68.4199L70.9775 68.3789V67.9756C70.9775 67.4925 70.8636 67.1325 70.6357 66.8955C70.4124 66.654 70.0638 66.5332 69.5898 66.5332C69.2025 66.5332 68.8311 66.5902 68.4756 66.7041C68.1201 66.818 67.7783 66.9525 67.4502 67.1074L66.9307 65.959C67.3408 65.7448 67.7897 65.583 68.2773 65.4736C68.765 65.3597 69.2253 65.3027 69.6582 65.3027C70.6198 65.3027 71.3444 65.5124 71.832 65.9316C72.3242 66.3509 72.5703 67.0094 72.5703 67.9072V73H71.4219ZM69.0293 71.9062C69.6126 71.9062 70.0798 71.7445 70.4307 71.4209C70.7861 71.0928 70.9639 70.6348 70.9639 70.0469V69.3906L69.9932 69.4316C69.2367 69.459 68.6852 69.5866 68.3389 69.8145C67.9971 70.0378 67.8262 70.3818 67.8262 70.8467C67.8262 71.1839 67.9264 71.446 68.127 71.6328C68.3275 71.8151 68.6283 71.9062 69.0293 71.9062ZM85.1006 73H83.4873V68.3516C83.4873 67.7682 83.3688 67.333 83.1318 67.0459C82.8994 66.7588 82.528 66.6152 82.0176 66.6152C81.3385 66.6152 80.8418 66.8158 80.5273 67.2168C80.2129 67.6178 80.0557 68.29 80.0557 69.2334V73H78.4492V65.4395H79.707L79.9326 66.4307H80.0146C80.2425 66.0706 80.5661 65.7926 80.9854 65.5967C81.4046 65.4007 81.8695 65.3027 82.3799 65.3027C84.1937 65.3027 85.1006 66.2256 85.1006 68.0713V73ZM92.0664 73L91.7451 71.9473H91.6904C91.3258 72.4076 90.959 72.722 90.5898 72.8906C90.2207 73.0547 89.7467 73.1367 89.168 73.1367C88.4251 73.1367 87.8441 72.9362 87.4248 72.5352C87.0101 72.1341 86.8027 71.5667 86.8027 70.833C86.8027 70.0537 87.0921 69.4658 87.6709 69.0693C88.2497 68.6729 89.1315 68.4564 90.3164 68.4199L91.6221 68.3789V67.9756C91.6221 67.4925 91.5081 67.1325 91.2803 66.8955C91.057 66.654 90.7083 66.5332 90.2344 66.5332C89.847 66.5332 89.4756 66.5902 89.1201 66.7041C88.7646 66.818 88.4229 66.9525 88.0947 67.1074L87.5752 65.959C87.9854 65.7448 88.4342 65.583 88.9219 65.4736C89.4095 65.3597 89.8698 65.3027 90.3027 65.3027C91.2643 65.3027 91.9889 65.5124 92.4766 65.9316C92.9688 66.3509 93.2148 67.0094 93.2148 67.9072V73H92.0664ZM89.6738 71.9062C90.2572 71.9062 90.7243 71.7445 91.0752 71.4209C91.4307 71.0928 91.6084 70.6348 91.6084 70.0469V69.3906L90.6377 69.4316C89.8812 69.459 89.3298 69.5866 88.9834 69.8145C88.6416 70.0378 88.4707 70.3818 88.4707 70.8467C88.4707 71.1839 88.571 71.446 88.7715 71.6328C88.972 71.8151 89.2728 71.9062 89.6738 71.9062ZM91.3008 64.5166C91.1094 64.5166 90.9202 64.4779 90.7334 64.4004C90.5511 64.3184 90.3711 64.2295 90.1934 64.1338C90.0156 64.0335 89.8424 63.9447 89.6738 63.8672C89.5052 63.7852 89.3434 63.7441 89.1885 63.7441C88.8193 63.7441 88.571 64.0062 88.4434 64.5303H87.6094C87.6641 63.8968 87.8395 63.4137 88.1357 63.0811C88.4365 62.7484 88.8193 62.582 89.2842 62.582C89.4801 62.582 89.6715 62.623 89.8584 62.7051C90.0452 62.7826 90.2275 62.8714 90.4053 62.9717C90.583 63.0674 90.7539 63.1562 90.918 63.2383C91.0866 63.3158 91.2484 63.3545 91.4033 63.3545C91.7633 63.3545 92.0049 63.0924 92.1279 62.5684H92.9824C92.9277 63.179 92.7523 63.6576 92.4561 64.0039C92.1598 64.3457 91.7747 64.5166 91.3008 64.5166ZM102.163 69.2061C102.163 70.4411 101.846 71.4049 101.213 72.0977C100.579 72.7904 99.6976 73.1367 98.5674 73.1367C97.861 73.1367 97.2367 72.9772 96.6943 72.6582C96.152 72.3392 95.735 71.8812 95.4434 71.2842C95.1517 70.6872 95.0059 69.9945 95.0059 69.2061C95.0059 67.9801 95.3203 67.0231 95.9492 66.335C96.5781 65.6468 97.4645 65.3027 98.6084 65.3027C99.7021 65.3027 100.568 65.6559 101.206 66.3623C101.844 67.0641 102.163 68.012 102.163 69.2061ZM96.6602 69.2061C96.6602 70.9515 97.305 71.8242 98.5947 71.8242C99.8708 71.8242 100.509 70.9515 100.509 69.2061C100.509 67.4788 99.8662 66.6152 98.5811 66.6152C97.9066 66.6152 97.4167 66.8385 97.1113 67.2852C96.8105 67.7318 96.6602 68.3721 96.6602 69.2061ZM110.469 71.8379C110.861 71.8379 111.253 71.7764 111.645 71.6533V72.8633C111.467 72.9408 111.237 73.0046 110.954 73.0547C110.676 73.1094 110.387 73.1367 110.086 73.1367C108.564 73.1367 107.803 72.3346 107.803 70.7305V66.6562H106.771V65.9453L107.878 65.3574L108.425 63.7578H109.416V65.4395H111.569V66.6562H109.416V70.7031C109.416 71.0905 109.512 71.3776 109.703 71.5645C109.899 71.7467 110.154 71.8379 110.469 71.8379ZM116.464 73.1367C115.288 73.1367 114.368 72.7949 113.702 72.1113C113.041 71.4232 112.711 70.4775 112.711 69.2744C112.711 68.0394 113.019 67.0687 113.634 66.3623C114.249 65.6559 115.094 65.3027 116.17 65.3027C117.168 65.3027 117.956 65.6058 118.535 66.2119C119.114 66.818 119.403 67.652 119.403 68.7139V69.582H114.365C114.388 70.3158 114.586 70.8809 114.96 71.2773C115.334 71.6693 115.86 71.8652 116.539 71.8652C116.986 71.8652 117.4 71.8242 117.783 71.7422C118.171 71.6556 118.585 71.5143 119.027 71.3184V72.624C118.635 72.8109 118.239 72.943 117.838 73.0205C117.437 73.098 116.979 73.1367 116.464 73.1367ZM116.17 66.5195C115.66 66.5195 115.249 66.6813 114.939 67.0049C114.634 67.3285 114.452 67.8001 114.393 68.4199H117.824C117.815 67.7956 117.665 67.3239 117.373 67.0049C117.081 66.6813 116.68 66.5195 116.17 66.5195ZM127.6 73H125.986V68.3379C125.986 67.7591 125.877 67.3285 125.658 67.0459C125.439 66.7588 125.098 66.6152 124.633 66.6152C124.013 66.6152 123.557 66.818 123.266 67.2236C122.979 67.6247 122.835 68.2946 122.835 69.2334V73H121.229V65.4395H122.486L122.712 66.4307H122.794C123.004 66.0706 123.307 65.7926 123.703 65.5967C124.104 65.4007 124.544 65.3027 125.022 65.3027C126.185 65.3027 126.955 65.6992 127.333 66.4922H127.442C127.666 66.1185 127.98 65.8268 128.386 65.6172C128.791 65.4076 129.256 65.3027 129.78 65.3027C130.683 65.3027 131.339 65.5306 131.749 65.9863C132.164 66.4421 132.371 67.137 132.371 68.0713V73H130.765V68.3379C130.765 67.7591 130.653 67.3285 130.43 67.0459C130.211 66.7588 129.869 66.6152 129.404 66.6152C128.78 66.6152 128.322 66.8112 128.03 67.2031C127.743 67.5905 127.6 68.1875 127.6 68.9941V73ZM139.856 73H138.25V65.4395H139.856V73ZM138.154 63.4365C138.154 63.1494 138.232 62.9284 138.387 62.7734C138.546 62.6185 138.772 62.541 139.063 62.541C139.346 62.541 139.565 62.6185 139.72 62.7734C139.879 62.9284 139.959 63.1494 139.959 63.4365C139.959 63.71 139.879 63.9264 139.72 64.0859C139.565 64.2409 139.346 64.3184 139.063 64.3184C138.772 64.3184 138.546 64.2409 138.387 64.0859C138.232 63.9264 138.154 63.71 138.154 63.4365ZM144.977 71.8379C145.368 71.8379 145.76 71.7764 146.152 71.6533V72.8633C145.975 72.9408 145.744 73.0046 145.462 73.0547C145.184 73.1094 144.895 73.1367 144.594 73.1367C143.072 73.1367 142.311 72.3346 142.311 70.7305V66.6562H141.278V65.9453L142.386 65.3574L142.933 63.7578H143.924V65.4395H146.077V66.6562H143.924V70.7031C143.924 71.0905 144.02 71.3776 144.211 71.5645C144.407 71.7467 144.662 71.8379 144.977 71.8379ZM150.972 73.1367C149.796 73.1367 148.875 72.7949 148.21 72.1113C147.549 71.4232 147.219 70.4775 147.219 69.2744C147.219 68.0394 147.526 67.0687 148.142 66.3623C148.757 65.6559 149.602 65.3027 150.678 65.3027C151.676 65.3027 152.464 65.6058 153.043 66.2119C153.622 66.818 153.911 67.652 153.911 68.7139V69.582H148.873C148.896 70.3158 149.094 70.8809 149.468 71.2773C149.841 71.6693 150.368 71.8652 151.047 71.8652C151.493 71.8652 151.908 71.8242 152.291 71.7422C152.678 71.6556 153.093 71.5143 153.535 71.3184V72.624C153.143 72.8109 152.747 72.943 152.346 73.0205C151.945 73.098 151.487 73.1367 150.972 73.1367ZM150.678 66.5195C150.167 66.5195 149.757 66.6813 149.447 67.0049C149.142 67.3285 148.96 67.8001 148.9 68.4199H152.332C152.323 67.7956 152.173 67.3239 151.881 67.0049C151.589 66.6813 151.188 66.5195 150.678 66.5195ZM162.388 73H160.774V68.3516C160.774 67.7682 160.656 67.333 160.419 67.0459C160.187 66.7588 159.815 66.6152 159.305 66.6152C158.626 66.6152 158.129 66.8158 157.814 67.2168C157.5 67.6178 157.343 68.29 157.343 69.2334V73H155.736V65.4395H156.994L157.22 66.4307H157.302C157.53 66.0706 157.853 65.7926 158.272 65.5967C158.692 65.4007 159.157 65.3027 159.667 65.3027C161.481 65.3027 162.388 66.2256 162.388 68.0713V73ZM169.702 70.8467C169.702 71.585 169.433 72.1523 168.896 72.5488C168.358 72.9408 167.588 73.1367 166.585 73.1367C165.578 73.1367 164.769 72.984 164.158 72.6787V71.291C165.047 71.7012 165.874 71.9062 166.64 71.9062C167.629 71.9062 168.123 71.6077 168.123 71.0107C168.123 70.8193 168.068 70.6598 167.959 70.5322C167.85 70.4046 167.67 70.2725 167.419 70.1357C167.168 69.999 166.82 69.8441 166.373 69.6709C165.503 69.3337 164.912 68.9964 164.603 68.6592C164.297 68.3219 164.145 67.8844 164.145 67.3467C164.145 66.6995 164.404 66.1982 164.924 65.8428C165.448 65.4827 166.159 65.3027 167.057 65.3027C167.945 65.3027 168.786 65.4827 169.579 65.8428L169.06 67.0527C168.244 66.7155 167.558 66.5469 167.002 66.5469C166.154 66.5469 165.73 66.7884 165.73 67.2715C165.73 67.5085 165.84 67.709 166.059 67.873C166.282 68.0371 166.765 68.2627 167.508 68.5498C168.132 68.7913 168.586 69.0124 168.868 69.2129C169.151 69.4134 169.36 69.6458 169.497 69.9102C169.634 70.1699 169.702 70.4821 169.702 70.8467ZM180.161 70.8467C180.161 71.585 179.892 72.1523 179.354 72.5488C178.817 72.9408 178.047 73.1367 177.044 73.1367C176.037 73.1367 175.228 72.984 174.617 72.6787V71.291C175.506 71.7012 176.333 71.9062 177.099 71.9062C178.088 71.9062 178.582 71.6077 178.582 71.0107C178.582 70.8193 178.527 70.6598 178.418 70.5322C178.309 70.4046 178.129 70.2725 177.878 70.1357C177.627 69.999 177.279 69.8441 176.832 69.6709C175.962 69.3337 175.371 68.9964 175.062 68.6592C174.756 68.3219 174.604 67.8844 174.604 67.3467C174.604 66.6995 174.863 66.1982 175.383 65.8428C175.907 65.4827 176.618 65.3027 177.516 65.3027C178.404 65.3027 179.245 65.4827 180.038 65.8428L179.519 67.0527C178.703 66.7155 178.017 66.5469 177.461 66.5469C176.613 66.5469 176.189 66.7884 176.189 67.2715C176.189 67.5085 176.299 67.709 176.518 67.873C176.741 68.0371 177.224 68.2627 177.967 68.5498C178.591 68.7913 179.045 69.0124 179.327 69.2129C179.61 69.4134 179.819 69.6458 179.956 69.9102C180.093 70.1699 180.161 70.4821 180.161 70.8467ZM186.635 73L186.313 71.9473H186.259C185.894 72.4076 185.527 72.722 185.158 72.8906C184.789 73.0547 184.315 73.1367 183.736 73.1367C182.993 73.1367 182.412 72.9362 181.993 72.5352C181.578 72.1341 181.371 71.5667 181.371 70.833C181.371 70.0537 181.66 69.4658 182.239 69.0693C182.818 68.6729 183.7 68.4564 184.885 68.4199L186.19 68.3789V67.9756C186.19 67.4925 186.076 67.1325 185.849 66.8955C185.625 66.654 185.277 66.5332 184.803 66.5332C184.415 66.5332 184.044 66.5902 183.688 66.7041C183.333 66.818 182.991 66.9525 182.663 67.1074L182.144 65.959C182.554 65.7448 183.003 65.583 183.49 65.4736C183.978 65.3597 184.438 65.3027 184.871 65.3027C185.833 65.3027 186.557 65.5124 187.045 65.9316C187.537 66.3509 187.783 67.0094 187.783 67.9072V73H186.635ZM184.242 71.9062C184.826 71.9062 185.293 71.7445 185.644 71.4209C185.999 71.0928 186.177 70.6348 186.177 70.0469V69.3906L185.206 69.4316C184.45 69.459 183.898 69.5866 183.552 69.8145C183.21 70.0378 183.039 70.3818 183.039 70.8467C183.039 71.1839 183.139 71.446 183.34 71.6328C183.54 71.8151 183.841 71.9062 184.242 71.9062ZM191.632 73H190.025V62.3633H191.632V73ZM195.658 73L192.787 65.4395H194.482L196.021 69.835C196.285 70.5732 196.444 71.1702 196.499 71.626H196.554C196.595 71.2979 196.754 70.7008 197.032 69.835L198.57 65.4395H200.279L197.395 73H195.658ZM208.134 69.2061C208.134 70.4411 207.817 71.4049 207.184 72.0977C206.55 72.7904 205.668 73.1367 204.538 73.1367C203.832 73.1367 203.207 72.9772 202.665 72.6582C202.123 72.3392 201.706 71.8812 201.414 71.2842C201.122 70.6872 200.977 69.9945 200.977 69.2061C200.977 67.9801 201.291 67.0231 201.92 66.335C202.549 65.6468 203.435 65.3027 204.579 65.3027C205.673 65.3027 206.539 65.6559 207.177 66.3623C207.815 67.0641 208.134 68.012 208.134 69.2061ZM202.631 69.2061C202.631 70.9515 203.276 71.8242 204.565 71.8242C205.841 71.8242 206.479 70.9515 206.479 69.2061C206.479 67.4788 205.837 66.6152 204.552 66.6152C203.877 66.6152 203.387 66.8385 203.082 67.2852C202.781 67.7318 202.631 68.3721 202.631 69.2061ZM215.065 70.8467C215.065 71.585 214.797 72.1523 214.259 72.5488C213.721 72.9408 212.951 73.1367 211.948 73.1367C210.941 73.1367 210.132 72.984 209.521 72.6787V71.291C210.41 71.7012 211.237 71.9062 212.003 71.9062C212.992 71.9062 213.486 71.6077 213.486 71.0107C213.486 70.8193 213.432 70.6598 213.322 70.5322C213.213 70.4046 213.033 70.2725 212.782 70.1357C212.532 69.999 212.183 69.8441 211.736 69.6709C210.866 69.3337 210.276 68.9964 209.966 68.6592C209.66 68.3219 209.508 67.8844 209.508 67.3467C209.508 66.6995 209.768 66.1982 210.287 65.8428C210.811 65.4827 211.522 65.3027 212.42 65.3027C213.309 65.3027 214.149 65.4827 214.942 65.8428L214.423 67.0527C213.607 66.7155 212.921 66.5469 212.365 66.5469C211.518 66.5469 211.094 66.7884 211.094 67.2715C211.094 67.5085 211.203 67.709 211.422 67.873C211.645 68.0371 212.128 68.2627 212.871 68.5498C213.495 68.7913 213.949 69.0124 214.231 69.2129C214.514 69.4134 214.724 69.6458 214.86 69.9102C214.997 70.1699 215.065 70.4821 215.065 70.8467Z" fill="#989898"/>
<ellipse cx="100" cy="24" rx="25" ry="24" fill="#F2F2F2"/>
<path d="M107 33L100 28L93 33V17C93 16.4696 93.2107 15.9609 93.5858 15.5858C93.9609 15.2107 94.4696 15 95 15H105C105.53 15 106.039 15.2107 106.414 15.5858C106.789 15.9609 107 16.4696 107 17V33Z" stroke="#C4C4C4" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
