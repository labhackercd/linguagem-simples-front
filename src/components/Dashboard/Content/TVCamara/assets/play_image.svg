<svg width="50" height="47" viewBox="0 0 50 47" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M0 0H50V47H0V0Z" fill="white"/>
<path d="M25 37C32.1797 37 38 31.1797 38 24C38 16.8203 32.1797 11 25 11C17.8203 11 12 16.8203 12 24C12 31.1797 17.8203 37 25 37Z" fill="#F4F4F4"/>
<path d="M22.3994 18.8003L30.1994 24.0003L22.3994 29.2003V18.8003Z" stroke="#C4C4C4" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
