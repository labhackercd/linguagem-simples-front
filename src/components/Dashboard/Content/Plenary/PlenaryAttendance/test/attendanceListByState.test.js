import React from 'react';
import {shallow} from "enzyme/build";
import { mount} from 'enzyme';
import MockAdapter from "axios-mock-adapter"
import CircularProgress from '@material-ui/core/CircularProgress';
import AttendanceListByState from '../attendaceListByState'
import CongressPersonLine from '../congressPersonLine'
import {mockPresenceList} from './assets/mockdata'

describe('Testing AttendanceListByState Card', () => {

    it("should render the AttendanceListByState component and match snapshot ", () => {


      const component = mount(<AttendanceListByState plenaryAttendanceList={mockPresenceList}/>);
  
      expect(component.exists()).toEqual(true);
      expect(component).toMatchSnapshot();
    });
  
  
  });
  
