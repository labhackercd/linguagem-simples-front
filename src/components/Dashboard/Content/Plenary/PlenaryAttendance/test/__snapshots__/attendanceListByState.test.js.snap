// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing AttendanceListByState Card should render the AttendanceListByState component and match snapshot  1`] = `
<AttendanceListByState
  plenaryAttendanceList={
    Array [
      Object {
        "carteira": 190,
        "datInicioOrdemDia": null,
        "datInicioSessao": null,
        "datRegistroPresencaPlenario": null,
        "nomReduzido": "Abílio Santana    ",
        "nomSessao": null,
        "numLegislatura": 56,
        "numPosicaoUF": 16,
        "numSessao": 0,
        "sigPartido": "PL        ",
        "sigUF": "BA",
        "tipoPresenca": "A",
      },
      Object {
        "carteira": 332,
        "datInicioOrdemDia": null,
        "datInicioSessao": null,
        "datRegistroPresencaPlenario": null,
        "nomReduzido": "Abou Anni         ",
        "nomSessao": null,
        "numLegislatura": 56,
        "numPosicaoUF": 20,
        "numSessao": 0,
        "sigPartido": "PSL       ",
        "sigUF": "SP",
        "tipoPresenca": "A",
      },
      Object {
        "carteira": 10,
        "datInicioOrdemDia": null,
        "datInicioSessao": null,
        "datRegistroPresencaPlenario": null,
        "nomReduzido": "Acácio Favacho    ",
        "nomSessao": null,
        "numLegislatura": 56,
        "numPosicaoUF": 2,
        "numSessao": 0,
        "sigPartido": "PROS      ",
        "sigUF": "AP",
        "tipoPresenca": "A",
      },
      Object {
        "carteira": 191,
        "datInicioOrdemDia": null,
        "datInicioSessao": null,
        "datRegistroPresencaPlenario": null,
        "nomReduzido": "Adolfo Viana      ",
        "nomSessao": null,
        "numLegislatura": 56,
        "numPosicaoUF": 16,
        "numSessao": 0,
        "sigPartido": "PSDB      ",
        "sigUF": "BA",
        "tipoPresenca": "A",
      },
      Object {
        "carteira": 333,
        "datInicioOrdemDia": null,
        "datInicioSessao": null,
        "datRegistroPresencaPlenario": null,
        "nomReduzido": "Adriana Ventura   ",
        "nomSessao": null,
        "numLegislatura": 56,
        "numPosicaoUF": 20,
        "numSessao": 0,
        "sigPartido": "Novo      ",
        "sigUF": "SP",
        "tipoPresenca": "A",
      },
      Object {
        "carteira": 416,
        "datInicioOrdemDia": null,
        "datInicioSessao": null,
        "datRegistroPresencaPlenario": null,
        "nomReduzido": "Adriano do Baldy  ",
        "nomSessao": null,
        "numLegislatura": 56,
        "numPosicaoUF": 23,
        "numSessao": 0,
        "sigPartido": "PP        ",
        "sigUF": "GO",
        "tipoPresenca": "A",
      },
      Object {
        "carteira": 221,
        "datInicioOrdemDia": null,
        "datInicioSessao": null,
        "datRegistroPresencaPlenario": null,
        "nomReduzido": "Aécio Neves       ",
        "nomSessao": null,
        "numLegislatura": 56,
        "numPosicaoUF": 17,
        "numSessao": 0,
        "sigPartido": "PSDB      ",
        "sigUF": "MG",
        "tipoPresenca": "A",
      },
    ]
  }
>
  <Styled(MuiBox)
    alignItems="center"
    display="flex"
    justifyContent="center"
    minHeight="10vh"
    minWidth="40vh"
    width={1}
  >
    <div
      className="MuiBox-root MuiBox-root-1"
    >
      <WithStyles(ForwardRef(LinearProgress))>
        <ForwardRef(LinearProgress)
          classes={
            Object {
              "bar": "MuiLinearProgress-bar",
              "bar1Buffer": "MuiLinearProgress-bar1Buffer",
              "bar1Determinate": "MuiLinearProgress-bar1Determinate",
              "bar1Indeterminate": "MuiLinearProgress-bar1Indeterminate",
              "bar2Buffer": "MuiLinearProgress-bar2Buffer",
              "bar2Indeterminate": "MuiLinearProgress-bar2Indeterminate",
              "barColorPrimary": "MuiLinearProgress-barColorPrimary",
              "barColorSecondary": "MuiLinearProgress-barColorSecondary",
              "buffer": "MuiLinearProgress-buffer",
              "colorPrimary": "MuiLinearProgress-colorPrimary",
              "colorSecondary": "MuiLinearProgress-colorSecondary",
              "dashed": "MuiLinearProgress-dashed",
              "dashedColorPrimary": "MuiLinearProgress-dashedColorPrimary",
              "dashedColorSecondary": "MuiLinearProgress-dashedColorSecondary",
              "determinate": "MuiLinearProgress-determinate",
              "indeterminate": "MuiLinearProgress-indeterminate",
              "query": "MuiLinearProgress-query",
              "root": "MuiLinearProgress-root",
            }
          }
        >
          <div
            className="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate"
            role="progressbar"
          >
            <div
              className="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Indeterminate"
              style={Object {}}
            />
            <div
              className="MuiLinearProgress-bar MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorPrimary"
              style={Object {}}
            />
          </div>
        </ForwardRef(LinearProgress)>
      </WithStyles(ForwardRef(LinearProgress))>
    </div>
  </Styled(MuiBox)>
</AttendanceListByState>
`;
